okhttp/src/test/java/okhttp3/KotlinSourceCompatibilityTest.kt

```diff
@@ -308,9 +308,6 @@ class KotlinSourceCompatibilityTest {
     val maxRequestsPerHost: Int = dispatcher.maxRequestsPerHost
     dispatcher.maxRequestsPerHost = 0
     val executorService: ExecutorService = dispatcher.executorService()
-    dispatcher.setIdleCallback(object : Runnable {
```

Comment:
turns out to be quite difficult to provide a `ReplaceWith` for this; I think it might be an IntelliJ bug.